src/main/logic/Logic.cpp

```diff
@@ -345,6 +357,8 @@ void Logic::startNewGame(){
 	eventManager->fireEvent(gameStateChangedEvent);
 	//Now we need to initalise everything
 	generateLevel();
+	//Setup the players
+	numLives = 1;
```

Comment:
Lets move the number of lives out of the Logic and into the Player. This has been raised as part of issue #67.